templates/tools/doxygen/Doxyfile.php.include

```diff
@@ -0,0 +1,25 @@
+
+<%namespace import="doxy_base" file="Doxyfile.base"/>
+<%def name="gen_doxyfile(internal)">
+<%
+  import itertools
+  import glob
+  import os
+  import fnmatch
+  srcdoc = []
+  for dirpath, dirnames, filenames in os.walk('src/php'):
+    for filename in filenames:
+      if os.path.splitext(filename)[1] == '.md':
+        srcdoc.append(os.path.join(dirpath, filename))
+
+  version = settings.php_version
+  doxy_input = ' \\\n'.join(sorted(set(
+        itertools.chain(
+            glob.glob('src/php/lib/Grpc/*.php'),
+            glob.glob('src/php/lib/Grpc/Internal/*.php') if internal else [],
```

Comment:
You need to add those 7 "fake" .php files here, because, according to source control, those files don't exist in the `lib/Grpc/*.php` directory. As of now, sanity checks will fail because generating this file `templates/tools/doxygen/Doxyfile.php.include` into `tools/doxygen/Doxyfile.php` would _not_ have matched.

Reply:
I have combined the following 3 steps. Could you please re-run source control to see if it could pass?
step1, cd grpc/src/php, run ./bin/php_extension_to_php_doc.sh to generate fake PHP files.
step2, run grpc/tools/doxygen/run_doxygen.sh to generate html files into grpc/doc/ref/php
step3, delete fake PHP files from grpc/src/php/lib/Grpc.